docs/src/development/macos.md

```diff
@@ -72,6 +72,10 @@ cargo test --workspace
 
 ## Troubleshooting
 
+### Can't compile Zed
+
+Before reporting the issue, make sure that you have the latest rustc version with `rustup update`.
```

Comment:
This was unasked for, and totally incorrect: we use a `rust-toolchain.toml` file to determine which rustc version and components will be ised (and downloaded) by rustup when working with this repo.

Reply:
I'll remove it

Reply:
Though if this is incorrect, I think we should remove it from `linux.md` and `windows.md`, should we not?

Reply:
Yeah, I thought I've asked someone to do that before, but apparently overlooked...

Reply:
> Yeah, I thought I've asked someone to do that before, but apparently overlooked...

It's pretty trivial. I can nuke those sections in this PR if you'd like